src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.scss

```diff
@@ -190,14 +190,14 @@ $sidebar-width: 200px;
       li a {
         color: vv.$white;
         display: block;
-        font-size: 1.1em;
+        font-size: 16px;
```

Comment:
Sure, relative units are prefered. I used px because after using the em values I converted [here](http://pxtoem.com/) the size didn't change much. Bad calculation :)